source/reference/api/start.txt

```diff
@@ -486,3 +514,39 @@ Endpoint Protection
 
 .. |endpoint| replace:: ``start``
 .. include:: /includes/fact-api-endpoint
+
+.. _c2c-start-verifier:
+
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` provides an embedded verifier to
+determine whether data transfer from the source cluster to the
+destination was successful.
+
+The specific verification checks performed by the embedded
+verifier are set using the ``verificationMode`` parameter.
+
+The ``verificationMode`` parameter supports the following
+verifier modes:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Mode
+     - Description
+   * - ``full``
+     - Performs all available verification checks.
+
+       This is the default mode.
```

Comment:
The verifier is not on by default in 1.9

Reply:
Fixed